pandas/_libs/tslibs/dtypes.pyi

```diff
@@ -4,6 +4,7 @@ from enum import Enum
 #  are imported in tests.
 _attrname_to_abbrevs: dict[str, str]
 _period_code_map: dict[str, int]
+DEPR_ABBREVS: dict[str, str]
```

Comment:
I haven't tried this, but would `dict[str, UnitChoices]`? then maybe you wouldn't need the `# type: ignore` in the other file. no worries if not

Reply:
Thank you, I did as you suggested. It works like a charm.